tests/test_multidict.py

```diff
@@ -397,6 +408,17 @@ def test_or2(self, cls: Type[MutableMultiMapping[str]]) -> None:
 
         assert {"key", "key2"} == {"key2"} | d.keys()
 
+    def test_or_not_implemented(self, cls: Type[MutableMultiMapping[str]]) -> None:
+        d = cls([("key", "value1")])
+
+        with pytest.raises(TypeError, match=r"unsupported operand type(\(s\))? for \|"):
+            d.keys() | 1
```

Comment:
It's not `or` but a "bitwise or", no?

Reply:
`or` will not give you any error

Reply:
@a5r0n I know but the test is called incorrectly, then. That name implies that `or` is tested, not `|`.